location/lbs/activity/audioset/dataprocessing/audio_processing.py

```diff
@@ -0,0 +1,577 @@
+"""
```

Comment:
This file is too large, consider breaking down the file into separate files, an example below.

Consider the following files with the example methods to place there:
 - downloader.py for all methods related to downloading, including `chop_audio()`
 - audioset_helper.py for `get_label_dict()`, `parse_metadata()`

Since the methods will have changed context by being moved to a separate file, it might be useful
to rename some of the methods.

Reply:
Ok. Should I create a main.py to drive the whole process?